scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -786,4 +870,33 @@ class FossId internal constructor(
             mapOf(SCAN_CODE_KEY to scanCode, SCAN_ID_KEY to scanId, SERVER_URL_KEY to config.serverUrl)
         )
     }
+
+    /**
+     * Return the [PurlType] as determined from the given [url], or null if there is no match. An issue will be added to
+     * [issues] in this case.
```

Comment:
Do not make the last sentence separate. Otherwise it sounds as if an issue is added in any case. Just continue the sentence instead:

> ..., in which case an issue will be added...